run-tests.php

```diff
@@ -2748,7 +2748,7 @@ function run_test($php, $file, $env)
         }
 
         // write .sh
-        if (strpos($log_format, 'S') !== false && file_put_contents($sh_filename, <<<SH
```

Comment:
This is not right, as the string uses interpolation. Just moving this up into `$sh_script = <<<SH ... SH;` before the if should do it.

Reply:
You're completely right. I've overlooked it. So sorry! Will fix it right now!

Reply:
You're completely right. I've overlooked it. So sorry! Will fix it right now!